### PR TITLE
Back port changes to 2.3.x

### PR DESCRIPTION
**Pull request by leslietilling**
## Purpose of this pull request

This pull request (PR) is a "backport" for the following PRs in master (2.4.x):

- #786 
- #790 
- #806 
- #824 
- #825 

## Magento release version

- [ ] 2.4.x

   Specify a patch release number, if applicable:

- [x] 2.3.x

   Specify a patch release number, if applicable:

## Product editions

Is this update specific to a product edition?

- [ ] Magento Open Source only
- [ ] Magento Commerce only

Is this update specific to an installed feature extension

- [ ] B2B extension
- [ ] Other feature set, please specify:

### CODE REVIEW

File: src/stores/store-urls-custom-admin.md [hunk 1]
```diff
@@ -51,27 +51,62 @@ As a precaution, do not try to change the Admin URL by yourself unless you know
...
+1. Use one of the following methods to clear the Magento cache:
+
+   - On the _Admin_ sidebar, go to **System** > _Tools_ > **Cache Management**. Then, click **Flush Magento Cache**.
+   - On the server, execute the following:
+      ```terminal
+      php bin/magento cache:flush
+      ```
+
+   {:.bs-callout-info}
+   Note that the changes made using the Method 1 will have the priority over the changes made in the `app/etc/env.php` file.
```
Comment (erikmarr) [inline src/stores/store-urls-custom-admin.md:88]:
```suggestion
   Note that the changes made using Method 1 will have priority over the changes made in the `app/etc/env.php` file.
```

File: src/stores/store-urls-custom-admin.md [hunk 2]
```diff
@@ -51,27 +51,62 @@ As a precaution, do not try to change the Admin URL by yourself unless you know
...
+
+## Restore the default Admin URL and Admin Path
+
+In case you have set an invalid Admin URL or an Admin Path and lost access to the backend, there is a way to fix this from the command line.
```
Comment (erikmarr) [inline src/stores/store-urls-custom-admin.md:92]:
```suggestion
If you have set an invalid Admin URL or Admin Path and lost access to the backend, you can fix this from the command line.
```

**Review by erikmarr — COMMENTED** [review]
Looks good. Just minor suggestions.

File: src/stores/store-urls-custom-admin.md (hunk 1 above)
Comment (leslietilling) [reply src/stores/store-urls-custom-admin.md:88]: good catch... will have to go back and fix that in master

**Review by erikmarr — APPROVED** [review]

**Comment by leslietilling** [comment]
running tests

**Pull request merged**